src/pages/MatterPage/MatterPage.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { FC, useCallback } from "react";
+import { useParams } from "react-router-dom";
+import { useAppConfigContext } from "../../app";
+import useFetchData from "../../containers/FetchDataContainer/useFetchData";
+import FetchDataContainer from "../../containers/FetchDataContainer/FetchDataContainer";
+import { MatterContainer } from "../../containers/MatterContainer";
+
+import MatterStatus from "../../models/MatterStatus";
+import IndexedMatterGram from "../../models/IndexedMatterGram";
+import EventMinutesItem from "../../models/EventMinutesItem";
+import Event from "../../models/Event";
+import Person from "../../models/Person";
+import Vote from "../../models/Vote";
+
+import MatterStatusService from "../../networking/MatterStatusService";
+import EventService from "../../networking/EventService";
+import MatterSponsorService from "../../networking/MatterSponsorService";
+import VoteService from "../../networking/VoteService";
+
+interface MatterContainerType {
+  matterStatus: MatterStatus;
+  indexedMatterGrams: IndexedMatterGram[];
+  event?: Event;
+  sponsors: Person[];
+  votes?: Vote[];
+  legislationHistory: EventMinutesItem[];
+}
+
+const MatterPage: FC = () => {
+  // Get the app config context
+  const { firebaseConfig } = useAppConfigContext();
+  // Get the id the matter, provided the route is `matter/:id`
+  const { id } = useParams<{ id: string }>();
+  //TODO: change these to Promise.all
+
+  const fetchMatterStatus = useCallback(async () => {
+    const matterStatusService = new MatterStatusService(firebaseConfig);
+    const matterSponsorService = new MatterSponsorService(firebaseConfig);
+    const eventService = new EventService(firebaseConfig);
+
+    const matterSponsors = await matterSponsorService.getMatterSponsorByMatterId(id);
+    const sponsors = matterSponsors
+      .filter((matterSponsor) => {
+        return matterSponsor.person;
+      })
+      .map((matterSponsor) => {
+        return matterSponsor.person as Person;
+      });
+
+    const matterStatuses = await matterStatusService.getMatterStatusesByMatterId(id);
```

Comment:
Could we group lines 41 and 50 into one `Promise.all` so that these two pieces of data are fetched concurrently. It will speed up the load of the page by a little.